src/jit/compiler.h

```diff
@@ -5323,10 +5323,10 @@ class Compiler
 
     void fgAdjustForAddressExposedOrWrittenThis();
 
-    bool                        fgProfileData_ILSizeMismatch;
-    ICorJitInfo::ProfileBuffer* fgProfileBuffer;
-    ULONG                       fgProfileBufferCount;
-    ULONG                       fgNumProfileRuns;
+    bool                      fgProfileData_ILSizeMismatch;
+    ICorJitInfo::BlockCounts* fgBlockCounts;
+    DWORD                     fgBlockCountsCount;
```

Comment:
Why the change from ULONG to DWORD? DWORD is not commonly used in the JIT codebase for "count". In fact neither is, it's more common to use `unsigned`.

Reply:
I hate ULONG (because it is 32-bits in size), and DWORD is used by the original definition.

src/inc/corbbtprof.h
```
struct CORBBTPROF_BLOCK_DATA    // This struct is also defined by:  ICorJitInfo.BlockCounts
{	{
    DWORD                          ILOffset;	    DWORD                          ILOffset;
    DWORD                          ExecutionCount;	    DWORD                          ExecutionCount;
};	};

```

Reply:
I prefer UINT32 or unsigned